Tests/Common/Securities/Cryptos/CryptoTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NodaTime;
+using NUnit.Framework;
+using QuantConnect.Securities;
+using QuantConnect.Securities.Crypto;
+using System;
+
+namespace QuantConnect.Tests.Common.Securities.Cryptos
+{
+    [TestFixture]
+    public class CryptoTests
+    {
+        private static TimeKeeper TimeKeeper
+        {
+            get { return new TimeKeeper(DateTime.Now, new[] { TimeZones.NewYork }); }
+        }
+
+        [Test]
+        [TestCase("BTCUSD", "USD")]
```

Comment:
Maybe it would be good to add a test with some crypto whose quote currency isn't `USD`